.env.development

```diff
@@ -24,5 +24,5 @@ MAINTENANCE_MODE=false
 LOG_LEVEL=debug
 
 # Cryptor
-CRYPTOR_URL=http://cryptor-sidecar:26272
+CRYPTOR_URL=http://cryptor:26272
```

Comment:
This is the same as `main`, not too sure why's here (I also have latest main locally). Can we .gitignore this file?

Reply:
good pickup.